### PR TITLE
 qocamlbrowser for update lablqml

### PR DESCRIPTION
**Pull request by Kakadu**

### CODE REVIEW

**Comment by Kakadu** [comment]
Should be merged after https://github.com/ocaml/opam-repository/pull/10022

File: packages/qocamlbrowser/qocamlbrowser.0.2.9/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+authors: "Kakadu"
+build: [
+  ["./configure"]
+  [make]
+]
+remove: [
+  [make "uninstall"]
+]
+depends: [
+  "lablqt" {>= "0.5.1"}
```
Comment (yallop) [inline packages/qocamlbrowser/qocamlbrowser.0.2.9/opam]: Should this be
```
"lablqml" {>= "0.5.1"}
```
?

**Comment by dbuenzli** [comment]
Ping.

**Comment by Kakadu** [comment]
Something weird is happening here. It invokes old scripts on centos for conf-qt but the new ones has been merged already https://github.com/ocaml/opam-repository/blob/master/packages/conf-qt/conf-qt.5.2.1/opam

**Comment by yallop** [comment]
I think the datakit CI builds the branch directly rather than merging it into `master` first.  Could you rebase against `master` to pick up the new changes?

**Comment by Kakadu** [comment]
@yallop, I thought that I did it but apparently not....
Thanks.

**Pull request merged**